### PR TITLE
xproperty: add version 0.12.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **xproperty/0.12.0**

https://github.com/jupyter-xeus/xproperty/compare/0.11.0...0.12.0

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [x] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/xproperty/all/conanfile.py [hunk 1]
```diff
@@ -12,25 +14,56 @@ class XpropertyConan(ConanFile):
...
     url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jupyter-xeus/xproperty"
+    topics = ("observer", "traitlets", "header-only")
     package_type = "header-library"
     settings = "os", "arch", "compiler", "build_type"
     no_copy_source = True
 
+    @property
+    def _min_cppstd(self):
+        return "14" if Version(self.version) <= "0.12.0" else "17"
```
Comment (RubenRBS) [inline recipes/xproperty/all/conanfile.py:26]: This error is a bit suprising, as per [jupyter-xeus/xproperty@`0.11.0...0.12.0`#diff-33394812ba](https://github.com/jupyter-xeus/xproperty/compare/0.11.0...0.12.0#diff-33394812ba204689144fd2f80832db83853ba1cb32403edb4e15fe4893e675fd)L40 it seems like they expect to be able to use C++14? Did you get a chance to communicate this to upstream?

**Review by RubenRBS — COMMENTED** [review]
Thanks! Quick question about C++14 support, otherwise looks great as always :)

File: recipes/xproperty/all/conanfile.py (hunk 1 above)
Comment (toge) [reply recipes/xproperty/all/conanfile.py:26]: @RubenRBS 
Thanks for your comment!
I create the issue about it. https://github.com/jupyter-xeus/xproperty/issues/69

**Review by perseoGI — APPROVED** [review]
LGTM

**Review by perseoGI — APPROVED** [review]

**Review by AbrilRBS — APPROVED** [review]

**Pull request merged**